test/psxy/smallcirvec.sh

```diff
@@ -13,7 +13,7 @@ cat << EOF > t.txt
 EOF
 gmt psbasemap -Rg -JG0/0/4.5i -Bag -P -K -Xc > $ps
 while read lon lat length color plon plat just; do
-	echo $lon $lat $length | gmt psxy -R -J -W1p,$color -S=0.4i+j${just}+o${plon}/${plat}+h1 -G$color -O -K >> $ps
+	echo $lon $lat $length 0 | gmt psxy -R -J -W1p,$color -S=0.4i+j${just}+o${plon}/${plat}+h1 -G$color -O -K >> $ps
```

Comment:
The order of length and 0 is switched from your comment on the PR. This relates to your comment about improving the docs (https://github.com/GenericMappingTools/gmt/pull/6161#issuecomment-1002303114); from what I can tell documentation about how to set small circle characteristics is missing.

Reply:
Yes, I switched because that is the only thing that worked to make the test pass.